textattack/models/wrappers/pytorch_model_wrapper.py

```diff
@@ -0,0 +1,45 @@
+import torch
+
+import textattack
+
+from .model_wrapper import ModelWrapper
+
+
+class PyTorchModelWrapper(ModelWrapper):
+    """Loads a PyTorch model (`nn.Module`) and tokenizer."""
+
+    # TODO test training code before putting up PR
+    def __init__(self, model, tokenizer, batch_size=32):
+        if not isinstance(model, torch.nn.Module):
+            raise TypeError(
+                f"PyTorch model must be torch.nn.Module, got type {type(model)}"
+            )
+
+        self.model = model.to(textattack.shared.utils.device)
+        self.tokenizer = tokenizer
+        self.batch_size = batch_size
+
+    @property
+    def get_model_device(self):
+        if hasattr(self.model, "model"):
```

Comment:
can you add a comment explaining why you have to do this? Why do some models have model attributes?

Reply:
it's just a handy thing to have for pytorch models. I added it bc it was a helper that was already in the code. Since we don't use it, though, let's get rid of it.